backend/game_server/lib/helpers/vector.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Helpers.NotVector do defexception message: "One of arguments is not Vector" end
```

Comment:
you can recognize too long line in github PR - when your line take two lines in github PR preview it is a little too long. it's not very important, but not everybody have big screens (I have not in my laptop). 
a reason: when you login to linux server by shell protocol, you are limited to some line length. or by using `nano`/`vim`? generally, long lines make reading live code on servers troublesome